lib/htmltoword/xslt/images.xslt

```diff
@@ -0,0 +1,130 @@
+<xsl:stylesheet xmlns:wpc="http://schemas.microsoft.com/office/word/2010/wordprocessingCanvas"
+                xmlns:mo="http://schemas.microsoft.com/office/mac/office/2008/main"
+                xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+                xmlns:mv="urn:schemas-microsoft-com:mac:vml"
+                xmlns:o="urn:schemas-microsoft-com:office:office"
+                xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships"
+                xmlns:m="http://schemas.openxmlformats.org/officeDocument/2006/math"
+                xmlns:v="urn:schemas-microsoft-com:vml"
+                xmlns:wp14="http://schemas.microsoft.com/office/word/2010/wordprocessingDrawing"
+                xmlns:wp="http://schemas.openxmlformats.org/drawingml/2006/wordprocessingDrawing"
+                xmlns:w10="urn:schemas-microsoft-com:office:word"
+                xmlns:w="http://schemas.openxmlformats.org/wordprocessingml/2006/main"
+                xmlns:w14="http://schemas.microsoft.com/office/word/2010/wordml"
+                xmlns:w15="http://schemas.microsoft.com/office/word/2012/wordml"
+                xmlns:wpg="http://schemas.microsoft.com/office/word/2010/wordprocessingGroup"
+                xmlns:wpi="http://schemas.microsoft.com/office/word/2010/wordprocessingInk"
+                xmlns:wne="http://schemas.microsoft.com/office/word/2006/wordml"
+                xmlns:wps="http://schemas.microsoft.com/office/word/2010/wordprocessingShape"
+                xmlns:a="http://schemas.openxmlformats.org/drawingml/2006/main"
+                xmlns:a14="http://schemas.microsoft.com/office/drawing/2010/main"
+                xmlns:pic="http://schemas.openxmlformats.org/drawingml/2006/picture"
+                xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:WX="http://schemas.microsoft.com/office/word/2003/auxHint"
+                xmlns:aml="http://schemas.microsoft.com/aml/2001/core"
+                xmlns:pkg="http://schemas.microsoft.com/office/2006/xmlPackage"
+                xmlns:msxsl="urn:schemas-microsoft-com:xslt"
+                xmlns:ext="http://www.xmllab.net/wordml2html/ext"
+                xmlns:java="http://xml.apache.org/xalan/java"
+                xmlns:str="http://exslt.org/strings"
+                xmlns:func="http://exslt.org/functions"
+                xmlns:fn="http://www.w3.org/2005/xpath-functions"
+                mc:Ignorable="w14 w15 wp14 a14"
+                version="1.0"
+                exclude-result-prefixes="java msxsl ext w o v WX aml w10"
+                extension-element-prefixes="func">
+
+  <xsl:include href="./image_functions.xslt"/>
+
+  <xsl:template match="img" name="image">
```

Comment:
Can you change the match to be `match="img|body/img"` ? 

There's a bug caused by https://github.com/karnov/htmltoword/blob/master/lib/htmltoword/xslt/base.xslt#L44 which makes markup like the following one to fail. 

```html
<body>
  <img src="https://assets-cdn.github.com/images/modules/logos_page/Octocat.png" style="width: 250px; height: 100px">
</body>
```

I'm still trying to figure out why we needed that line...